cmd/crc/cmd/start.go

```diff
@@ -203,3 +200,54 @@ func checkIfNewVersionAvailable(noUpdateCheck bool) {
 	}
 	logging.Debugf("No new version available. The latest version is %s", newVersion)
 }
+
+const startTemplate = `Started the OpenShift cluster.
+
+The server is accessible via web console at:
+  {{ .ClusterConfig.WebConsoleURL }}
+
+Log in as administrator:
+  Username: {{ .ClusterConfig.AdminCredentials.Username }}
+  Password: {{ .ClusterConfig.AdminCredentials.Password }}
+
+Log in as user:
+  Username: {{ .ClusterConfig.DeveloperCredentials.Username }}
+  Password: {{ .ClusterConfig.DeveloperCredentials.Password }}
+
+Use the 'oc' command line interface:
+  {{ .CommandLinePrefix }} {{ .EvalCommandLine }}
+  {{ .CommandLinePrefix }} oc login {{ .ClusterConfig.URL }}
+`
+
+type templateVariables struct {
+	ClusterConfig     *clusterConfig
+	EvalCommandLine   string
+	CommandLinePrefix string
+}
+
+func writeTemplatedMessage(writer io.Writer, s *startResult) error {
+	parsed, err := template.New("template").Parse(startTemplate)
+	if err != nil {
+		return err
+	}
+
+	return parsed.Execute(writer, &templateVariables{
+		ClusterConfig:     s.ClusterConfig,
+		EvalCommandLine:   shell.GenerateUsageHint(shellForTemplate(), "crc oc-env"),
+		CommandLinePrefix: commandLinePrefix(),
+	})
+}
+
+func commandLinePrefix() string {
+	if runtime.GOOS == "windows" {
+		return "PS>"
+	}
+	return "$"
+}
+
+func shellForTemplate() string {
+	if runtime.GOOS == "windows" {
+		return "powershell"
+	}
+	return ""
+}
```

Comment:
Wouldn't these 2 helpers be a better fit in the `shell` package? I though on Windows you could either use `powershell` or `cmd.exe`? Why `shellForTemplate()` when there is `shell.GetShell("")`?

Reply:
If the user launches it with cmd, the start output is really ugly: 
`@FOR /f "tokens=*" %%i IN ('crc oc-env') DO @call %%i`

I prefer to encourage the user to use PowerShell.

Reply:
> I prefer to encourage the user to use PowerShell.

But this is not what this PR is about, and the current PR output won't work if I ran crc from cmd.exe and c&p it